testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/transaction/mdb/timeout/TransactionTimeoutQueueSetupTask.java

```diff
@@ -63,10 +72,32 @@ public void tearDown(ManagementClient managementClient, String containerId) thro
                 adminOperations.removeJmsQueue(ANNOTATION_TIMEOUT_QUEUE_NAME);
                 adminOperations.removeJmsQueue(PROPERTY_TIMEOUT_QUEUE_NAME);
                 adminOperations.removeJmsQueue(REPLY_QUEUE_NAME);
+
+                removeAddressSettings(DEFAULT_TIMEOUT_QUEUE_NAME);
+                removeAddressSettings(PROPERTY_TIMEOUT_QUEUE_NAME);
             } finally {
                 adminOperations.close();
             }
         }
     }
 
+    // /subsystem=messaging-activemq/server=default/address-setting=queue.propertyTimeoutQueue:add(max-delivery-attempts=1)
+    private void setMaxDeliveryAttempts(String queueName) throws IOException {
+        ModelNode address = amqServerDefaultAdress.add("address-setting", "queue." + queueName);
```

Comment:
This add method mutates the target ModelNode, so you need to clone it before adding if you're using a shared object.